### ISSUE TITLE
a11y: Change number input (spinbox) to a normal input

### ISSUE THREAD

**Issue by pearlyong** [opengovsg/FormSG#5500]
**a11y users**
Mac Voiceover: reads out 0%, cannot access spinner, only vertical splitter
Talkback: cannot access spinner
iPhone Voiceover: '0.00 adjustable, required, swipe up or down with one finger to adjust the value' when you swipe up and down the values are '45035997000... 5% required'

**Sighted users**
Some of the users inadvertently increased/ decreased the value in the field when they scrolled up/ down the page with their mouse. Their mouse was most likely hovering over the field. 

**Proposal**
Remove number spinner altogether and make it a normal edit field (same as Angular). Initially we added the spinner and scrolling functionality to make it easier for users but seems like it's causing some unintended issues.

**Comment by mantariksh** [comment]
@pearlyong @staceytan1998 I'll remove the spinner and scrolling but unfortunately that doesn't fix the '45035997000... 5% required' issue on iPhone. will have to investigate that further.

**Comment by staceytan1998** [comment]
**Issue**
On Q5 (how many times you go to library in a month), the screen reader focus indicator cannot land on the + or - buttons to increase or decrease the number.

This seems to be an intentional design decision as the + / - buttons have aria-hidden = ""true"" attribute.

This is not technically an accessibility issue under WCAG. However, there is a disconnect between what a user can visually see and what a screen reader can access (i.e. you can visibly see the button, but the screen reader cannot focus on the button).

This can potentially cause confusion and frustration for screen reader users who are still partially sighted, as they may incorrectly think that their screen reader is not working properly due to the inability to focus on the buttons, when this was actually an intentional design decision during development.


**Suggestion**
Either one of the following should be considered:

1) Removing the +/- buttons entirely. While this is useful for incrementing and decrementing numbers in small quantities, this could potentially also be achieved by using a number picker. Alternatively, the HTML default <input type = ""number""> field works fine as well.

2) Enabling the +/- buttons to screen readers so they can be focused on (i.e. aria-hidden = ""false""). However, this comes with the consideration of whether it creates undesirable extra steps for the screen reader user to have to swipe / Tab during navigation (which might be the original itention for enabling aria-hidden = ""true"").
